docs/docs/how-tos/configure-gitops.md

```diff
@@ -0,0 +1,75 @@
+---
+id: configure-gitops
+title: Setup Nebari GitOps
+description: Setting up GitOps for Nebari
+---
+
+# Setup Nebari gitops
+
+## What is GitOps / Infrastructure as Code (IaC)?
+
+GitOps also known as Infrastructure as Code(Iac) is the practice of using git for
+managing the state of your infrastructure. This allows for easy
+auditing and discovering the current state of your system.
+
```

Comment:
```suggestion
In other parts of these docs, we covered [installing Nebari](/get-started/installing-nebari) and the various [methods of deployment](/get-started/deploy) which including walking you through the creation of the central configuration file for Nebari, `nebari-config.yaml`. We can take our automation one step further by ensuring that any changes to this configuration will redeploy the entire Nebari platform via CI. This document will show you how to set things up. 
```
Not sure about those link paths, and I didn't spend time making the wording right, but you get the point. We need something here to say "this is the icing on the cake". 

And as a sister comment... We need to link to this document at the bottom of each `Deploy Nebari on ...` doc as a `Next Steps`.